health-services/project-factory/src/server/utils/projectCampaignUtils.ts

```
@@ -0,0 +1,565 @@
+
+    // Filter by active status if specified
+    if (searchActiveOnly) {
+        query += ` AND isactive = true`;
+    }
+
+    try {
+        const queryResponse = await executeQuery(query, values);
+
+        if (fetchCountOnly) {
+            // Return the count as an integer if fetchCountOnly is true
+            return parseInt(queryResponse.rows[0].count, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)